include/freerdp/freerdp.h

```diff
@@ -240,7 +240,9 @@ struct rdp_freerdp
 								   Callback for context deallocation
 								   Can be set before calling freerdp_context_free() to have it executed before deallocation.
 								   Must be set to NULL if not needed. */
-	UINT64 paddingC[48 - 35]; /* 35 */
+	UINT64 paddingC[47 - 35]; /* 35 */
+
+	ALIGN64 ConnectionCallbackState; /* 48 */
```

Comment:
So the offset is 47, isn't it ? (IIRC the number in the comment is used by some scripts)

Reply:
not having a variable name here is weird isn't it ?